extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -28,7 +28,7 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
 
   private currentlySavingDraft = false;
   private disableSendingDrafts = false;
-  private saveDraftInterval?: number;
+  private saveDraftAlarm?: string;
```

Comment:
I think we still can use `setInterval` and `setTimeout` in content script code, `chrome.alarms` should be used only for background timers in service worker (for example for deleting expired items in in-memory storage - `Catch.setHandledInterval(() => inMemoryStore.deleteExpired(), 60000);`)